dmutils/formats.py

```diff
@@ -1,2 +1,39 @@
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 DATE_FORMAT = "%Y-%m-%d"
+
+LOTS = [
+    {
+        'lot': 'saas',
+        'lot_case': 'SaaS',
+        'label': u'Software as a Service',
```

Comment:
Labels are the only ones that end up in the template and I think that caused unicode errors on py3, but I don't remember what the exact issue was.